README.md

```diff
@@ -0,0 +1,32 @@
+# KCP API Catalog
+
+A library for kcp API catalog code.
+
+The API Catalog provides publicly available interfaces (APIs, code, CLIs) that allows users to curate a collection of available APIs (`APIExport`) in the KCP environment.
+
+## What is API Catalog?
+
+Each available API is encapsulated in a new API called `CatalogEntry` that is created by the user in a specific workspace.
+
+The `CatalogEntry` contains an identifier that links to its corresponding `APIExport`. Ideally, the identifier is a hash of `APIExport` name and its workspace location. This hash mechanism is utilized to conceal the direct identifier of `APIExport` which shouldn't be exposed to consumers for security reason. However, at the moment, `CatalogEntry` simply uses a list of `ExportReference` in the spec until the hash identifier is fully implemented.
```

Comment:
Couple comments:

- exports already have identities - what about is "not fully implemented"
- the identity hash is not a hash of the workspace path and name, it's a hash of the secret identity value
- this reads like we are making the catalog:export relationship 1:N only temporarily, but IIRC it's the end design, no?

Reply:
1. The original idea of identity here is a hash of the workspace path and name. I'm not sure if it was a new idea from Stefan or he indeed meant `IdentityHash`. Either way, I should remove all of future ideas given they may confuse people. Plus, they may not end up being what we are going to do. It's best to discuss what we do in this PR.